tests/validation/tests/v3_api/common.py

```diff
@@ -1170,9 +1170,11 @@ def delete_cluster(client, cluster):
             assert len(ip_filter) > 0
             print(ip_filter)
             aws_nodes = AmazonWebServices().get_nodes(filters)
+            if aws_nodes is None:
+                aws_nodes = AmazonWebServices().get_nodes(filters=[ip_filter])
```

Comment:
Could we add a check here if nodes are found?

Reply:
assert  len(aws_nodes) > 0

Reply:
@soumyalj an assertion may break the workflow. Instead, it will just print no instance is found in AWS and skip deleting instances, and go to the next step.